lib/typescript/typings/index.d.ts

```diff
@@ -41,7 +41,7 @@ export interface ReactNativeZoomableViewProps {
     zoomAnimatedValue?: Animated.Value;
     panAnimatedValueXY?: Animated.ValueXY;
     debug?: boolean;
-    onLayoutMeasured?: (size: Size2D & Vec2D) => void;
+    onLayout?: (size: Size2D & Vec2D) => void;
```

Comment:
I think this is cool, but it'll be different than the `ViewProps` version of `onLayout` because it's not keyed by `event`. So this will still be a breaking change. It's also not called when `onLayout` is called in the view component.

```
onLayout?: ((event: LayoutChangeEvent) => void) | undefined;
```

Is it possible to synthesize an event here somehow? Or force the `onLayout` event? Or just return `{ nativeEvent: { layout } }` and fake it?